data/beaconrestapi/src/integration-test/java/tech/pegasys/artemis/beaconrestapi/beacon/GetStateIntegrationTest.java

```diff
@@ -42,6 +48,50 @@ public void shouldReturnNoContentIfStoreNotDefined_queryBySlot() throws Exceptio
     assertNoContent(response);
   }
 
+  @Test
+  public void shouldReturnNoContentIfHeadRootMissing_queryBySlot() throws Exception {
```

Comment:
nit: most of the time function name, then should, like `queryBySlot_shouldReturnNoContentIfHeadRootMissing`

Reply:
There are 2 styles of test names that I typically use / have seen:
- methodName_variationA_variationB
- shouldDoX_variationA

I prefer the first style, but these tests are just invoking an http endpoint, so there's not really a method name to use.